packages/web/src/scss/components/UNSTABLE_ProductLogo/_UNSTABLE_ProductLogo.scss

```diff
@@ -0,0 +1,18 @@
+@use 'sass:map';
+@use 'theme' as theme;
+@use '../../tools/breakpoint';
+
+.UNSTABLE_ProductLogo {
+    width: fit-content;
+}
+
+.UNSTABLE_ProductLogo > img,
```

Comment:
If we count with `svg` and `img` as well, could you please mention it in the README.md? 🙏🏻

Reply:
Added